python/fromdocs/connection/login.py

```diff
@@ -0,0 +1,43 @@
+
+from grpc import RpcError
+from immudb import ImmudbClient
+
+URL = "localhost:3322"  # immudb running on your machine
+LOGIN = "immudb"        # Default username
+PASSWORD = "immudb"     # Default password
+DB = b"defaultdb"       # Default database name (must be in bytes)
+
+
+def main():
+    client = ImmudbClient(URL)
+    # database parameter is optional
+    client.login(LOGIN, PASSWORD, database=DB)
+    client.logout()
+
+    # Bad login
+    try:
+        client.login("verybadlogin", "verybadpassword")
+    except RpcError as exception:
+        print(exception.debug_error_string())
+        print(exception.details())
+
+
+    # Managed session support
+    with client.openManagedSession(LOGIN, PASSWORD, database=DB) as session:
+        transaction = session.newTx()
```

Comment:
for login/logout there were no actions, here it's executing sql stmts, we may keep just the session handling?